remotes/docker/pusher.go

```diff
@@ -338,6 +339,7 @@ func (pw *pushWriter) Commit(ctx context.Context, size int64, expected digest.Di
 	if resp == nil {
 		return errors.New("no response")
 	}
+	defer resp.Body.Close()
```

Comment:
Changed from `resp.Response.Body.Close()`, because in this branch `resp` is a regular http.Response

Reply:
yeah, that looks correct